README.rst

```diff
@@ -39,9 +39,9 @@ Requirements
 
 To be able to run **djoser** you have to meet following requirements:
 
-- Python (3.5, 3.6, 3.7, 3.8)
-- Django (1.11, 2.2, 3.1)
```

Comment:
I don't know if dropping support for Django 1.11 is a good idea. Sounds nice but it doesn't cost us too much to support it and plenty of people are still using it. I know DRF dropped support for 1.11 but their cost is bigger.

Reply:
Djoser is a glue layer between Django and DRF. If DRF does not support Django 1.11 (which lifecycle has ended BTW), why should Djoser?

Reply:
I didn't know it's officially finished. Then I agree.